src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -59,6 +59,24 @@ public static void GetDirectoryName_SpaceOrControlCharsThrowOnWindows(string pat
             }
         }
 
+        [Theory,
+            InlineData(" "),
+            InlineData("\r\n")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public static void GetDirectoryName_SpaceOrControlCharsThrowOnWindows_Core(string path)
+        {
+            Action action = () => Path.GetDirectoryName(path);
+            if (PlatformDetection.IsWindows && path.Equals(" "))
```

Comment:
This test is strange.  So " " goes to the "These are valid paths on Unix" path?  There are only two inputs... why not just have two different tests?